src/runtime-rs/src/shim-proto/examples/shim-proto-ttrpc-client.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019 Ant Financial
+// Copyright (c) 2021 Ant Group
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
```

Comment:
In the rest of Kata we're using SPDX identifiers rather than full license banners.

Reply:
Fixed in the secode revision